packages/search/src/spectrum-search.css

```diff
@@ -12,127 +12,424 @@ governing permissions and limitations under the License.
+        var(--spectrum-search-border-color-focus-hover)
+    );
+    --mod-textfield-border-color-keyboard-focus: var(
+        --mod-search-border-color-key-focus,
+        var(--spectrum-search-border-color-key-focus)
+    );
+    --mod-textfield-border-color-disabled: var(
+        --mod-search-border-color-disabled,
+        var(--spectrum-search-border-color-disabled)
+    );
+    --mod-textfield-background-color: var(
+        --mod-search-background-color,
+        var(--spectrum-search-background-color)
+    );
+    --mod-textfield-background-color-disabled: var(
+        --mod-search-background-color-disabled,
+        var(--spectrum-search-background-color-disabled)
+    );
+}
+.spectrum-Search--sizeS {
+    --spectrum-search-block-size: var(--spectrum-component-height-75);
+    --spectrum-search-icon-size: var(--spectrum-workflow-icon-size-75);
+    --spectrum-search-text-to-icon: var(--spectrum-text-to-visual-75);
+}
+.spectrum-Search--sizeL {
+    --spectrum-search-block-size: var(--spectrum-component-height-200);
+    --spectrum-search-icon-size: var(--spectrum-workflow-icon-size-200);
+    --spectrum-search-text-to-icon: var(--spectrum-text-to-visual-200);
+}
+.spectrum-Search--sizeXL {
+    --spectrum-search-block-size: var(--spectrum-component-height-300);
+    --spectrum-search-icon-size: var(--spectrum-workflow-icon-size-300);
+    --spectrum-search-text-to-icon: var(--spectrum-text-to-visual-300);
+}
+:host([quiet]) {
     --spectrum-search-quiet-button-offset: calc(
-        var(
-                --spectrum-actionbutton-m-texticon-min-width,
-                var(--spectrum-global-dimension-size-400)
-            ) / 2 - var(--spectrum-alias-ui-icon-cross-size-100) / 2
+        var(--mod-search-block-size, var(--spectrum-search-block-size)) / 2 -
+            var(
+                --mod-workflow-icon-size-100,
+                var(--spectrum-workflow-icon-size-100)
+            ) / 2
+    );
+    --spectrum-search-background-color: transparent;
+    --spectrum-search-background-color-disabled: transparent;
+    --spectrum-search-border-color-disabled: var(
+        --spectrum-disabled-border-color
     );
 }
+:host([quiet]) #textfield {
+    --spectrum-search-border-radius: 0;
+    --spectrum-search-edge-to-visual: var(
+        --spectrum-field-edge-to-visual-quiet
+    );
+}
+@media (forced-colors: active) {
+    #textfield #textfield,
+    #textfield #textfield .input {
+        --highcontrast-search-color-default: CanvasText;
+        --highcontrast-search-color-hover: CanvasText;
+        --highcontrast-search-color-focus: CanvasText;
+        --highcontrast-search-color-disabled: GrayText;
+    }
+    #textfield #button {
+        inset-block-start: calc(
+            var(--mod-search-border-width, var(--spectrum-search-border-width)) +
+                1px
+        );
+        inset-inline-end: calc(
+            var(--mod-search-border-width, var(--spectrum-search-border-width)) +
+                1px
+        );
```

Comment:
This seems to alter the location of the close button in an unexpected manner: https://8f8fe4262b9909dbf404d8774ebf5fad--spectrum-web-components.netlify.app/review/#SearchStories/focusedOverflowing.png

Reply:
This should be fixed in the latest update. I simplified the CSS related to that high contrast mode fix. Please take a look. I'm not seeing that screenshot in the VRTs anymore.